src/libraries/System.Net.Requests/src/System/Net/FileWebRequest.cs

```diff
@@ -36,6 +36,7 @@ internal FileWebRequest(Uri uri)
         }
 
         [Obsolete("Serialization is obsoleted for this type. https://go.microsoft.com/fwlink/?linkid=14202")]
+        [Obsolete(Obsoletions.WebRequestMessage, DiagnosticId = Obsoletions.WebRequestDiagId, UrlFormat = Obsoletions.SharedUrlFormat)]
```

Comment:
HttpClient does not support file URIs

Reply:
Do we have an alternative api for fie URIs?

It will warn even I remove this line if the FileWebRequest is created by `WebRequest.Create()` / `CreateDefault`.

Reply:
The closest API we have is for a user to call `Uri.LocalPath` and use `File.Open`. It should still be obsoleted, but a message of "use HttpClient" doesn't make sense.